content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,259 @@
+- :guilabel:`Type`: From the drop-down menu, select either :guilabel:`Experience`,
+  :guilabel:`Education`, :guilabel:`Internal Certification`, :guilabel:`Internal Training`, or type
+  in a new entry.
+- :guilabel:`Display Type`: Select either :guilabel:`Classic`, :guilabel:`Certification`, or
+  :guilabel:`Course` from the drop-down menu.
+- :guilabel:`Date Start` and :guilabel:`Date End`: Enter the start and end dates for the work
+  experience. To select a date, use the left and right arrows to scroll to the desired month, then
+  click on the day to select it.
+- :guilabel:`Description`: Enter any relevant details in the description field.
+
+Skills
+~~~~~~
+
+An employee's skills can be entered in the :guilabel:`Resume` tab in the same manner a resume line
+is created. Click the :guilabel:`Create a New Entry` button under *Skills* and a :guilabel:`Create
+Skills` form appears. Fill in the information, then click the :guilabel:`Save & Close` button if
+there is only one entry to add, or click the :guilabel:`Save & New` button to save the current entry
+and immediately create a new entry.
+
+.. image:: new_employee/create-skill.png
+   :align: center
+   :alt: Create a new skill for the employee.
+
+- :guilabel:`Skill Type`: Select from the drop-down menu either :guilabel:`Languages`,
+  :guilabel:`Dev`, :guilabel:`Music`, :guilabel:`Marketing`, or type in a new skill type. After
+  entering the new skill type, an option to :guilabel:`Create` the skill or :guilabel:`Create and
+  Edit` the skill appears. Click :guilabel:`Create and Edit`, and a pop-up for the specific skill
+  type appears. This can also be accessed with the :guilabel:`External Link` icon next to the new
+  skill. This form allows for the creation of specific skills and levels. Click :guilabel:`Add a
+  line` and enter the information for the new skill, then repeat for all other added skills. Repeat
+  this process for the :guilabel:`Levels` section. Click :guilabel:`Add a line` to add each level
+  and progress.
+
+  .. image:: new_employee/new-skills.png
+     :align: center
+     :alt: Add a new skill and levels.
+
+  .. example::
+     To add a math skill set, enter `Math` in the :guilabel:`Skill Type` field. In the
+     :guilabel:`Skills` field, enter `Algebra`, `Calculus`, and `Trigonometry`. And, in the
+     :guilabel:`Skill Level` field enter `beginner`, `intermeidate`, and `expert`. Then, either
+     click :guilabel:`Save & Close` of :guilabel:`Save & New`.
+
+- :guilabel:`Skill`: The corresponding :guilabel:`Skills` associated with the selected
+  :guilabel:`Skill Type` appear in a drop-down menu. For example, selecting *language* as the
+  :guilabel:`Skill Type` presents a variety of languages to select from under the
+  :guilabel:`Skills` field. Select the appropriate pre-configured skill, or type in a new one.
+- :guilabel:`Skill Level`: Pre-defined :guilabel:`Skill Levels` associated with the selected skill
```

Comment:
I agree- I forgot the pop-up only says Levels for the section, whereas on the individual skill section, when you add it, you see "skill level". So I believe your edits here are correct (I made these edits that is).

I originally put in Skill Level since that's what it appeared like in the previous screen, but this is more accurate..